README.md

```diff
@@ -1,9 +1,54 @@
 # PipelineDP
 
-PipelineDP is project for performing Differentially Private (DP) aggregations in Python Data Pipelines.
+PipelineDP is a framework for applying differential privacy to large
+datasets using batch processing systems such as Apache Spark, Apache Beam,
+and more.
+
+To make differential privacy accessible to non-experts, PipelineDP:
+
+* provides a convenient API familiar to Spark or Beam developers
+* encapsulates the complexities of differential privacy, such as
+* protection of outliers and rare categories, generation of safe
+  noise and privacy budget accounting;
+* supports many standard computations, such as count, sum, average, and is easily extensible to support other aggregation types.
 
 The project is in the early development stage. More description will be added later.
 
+## Getting started
+
+Take a look on examples of how to run PipelineDP Apache Spark or Beam:
+
+* [Apache Spark example](examples/movie_view_ratings_spark.py)
+* [Apache Beam example](examples/movie_view_ratings_beam.py)
+* [Local execution example](examples/movie_view_ratings.py)
+
+Here's a code sample for introducing how the private processing code on Spark looks like. For a more complete guide,
+please take a look on the examples above, or on the API guide.
+```python
+# Define the privacy budget available for our computation.
+budget_accountant = pipeline_dp.NaiveBudgetAccountant(total_epsilon=1,
+                                                      total_delta=1e-6)
+
+# Wrap Spark's RDD into it's private version. You will use this private wrapper
+# for all further processing instead of the Spark's RDD. Using the wrapper ensures
+# that only private statistics can be released.
+private_movie_views = \
+    make_private(movie_views, budget_accountant, lambda mv: mv.user_id)
+
+# Calculate the private sum
+dp_result = private_movie_views.sum(
+    SumParams(max_partitions_contributed=2,
+              max_contributions_per_partition=2,
+              low=1,
+              high=5,
+              partition_extractor=lambda mv: mv.movie_id,
+              value_extractor=lambda mv: mv.rating))
```

Comment:
Nit: add comment to this, something like, specifies what to sum

Reply:
Done.